src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1038,11 +1038,11 @@ julia> filter(row -> row.x > 1, df)
 
 julia> filter(row -> row["x"] > 1, df)
 2×2 DataFrame
- Row │ x      y      
+ Row │ x      y
      │ Int64  String
 ─────┼───────────────
    1 │     3  b
-   2 │     2  a                                                                    
+   2 │     2  a
```

Comment:
@nathanrboyer - for your future PRs can you please make sure to set your editor to trim trailing spaces? Thank you! (here is an example of a fix to your recent PR)

Reply:
Okay, I see.